From ede448c20122ba776ea7b68db7cff44efc0131de Mon Sep 17 00:00:00 2001
From: =?utf-8?q?J=C3=B6rg=20Sommer?= <joerg@alea.gnuu.de>
Date: Thu, 21 Aug 2008 14:43:01 +0200
Subject: [PATCH 3/5] Print a warning if downloading a log a second time

Some packages were built by more than one buildd. Tell the user about it.

Example:
http://buildd.debian-ports.org/fetch.php?&pkg=slrn&ver=0.9.8.1pl1-28&arch=armel
http://experimental.debian.net/fetch.php?&pkg=slrn&ver=0.9.8.1pl1-28&arch=armel
---
 getbuildlog |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/getbuildlog b/getbuildlog
index 7b4a781..fc70317 100755
--- a/getbuildlog
+++ b/getbuildlog
@@ -90,6 +90,9 @@ getbuildlog() {
         arch=${match##*arch=}
         arch=${arch%%&*}
 	match=`echo $match | sed -e 's/\+/%2B/g'`
+        if [ -e "${PACKAGE}_${ver}_${arch}.log" ]; then
+            echo "Found ${PACKAGE}_${ver}_${arch}.log a second time on $BASE"
+        fi
         wget -O "${PACKAGE}_${ver}_${arch}.log" "$BASE/$match&file=log"
     done
 
-- 
1.5.6.3

